src/autoplex/data/common/jobs.py

```diff
@@ -596,44 +596,50 @@ def collect_dft_data(
     isolated_atom_energies = {}
 
     for i, val in enumerate(dirs):
-        if os.path.exists(os.path.join(val, "vasprun.xml.gz")):
 
+        # TODO: think about fall back when we use ml potential outputs instead
+        has_vasp_output = os.path.exists(os.path.join(val, "vasprun.xml.gz"))
+        has_ase_output = os.path.exists(os.path.join(val, "final_atoms_object.xyz"))
+        if has_vasp_output:
             converged = check_convergence_vasp(os.path.join(val, "vasprun.xml.gz"))
-
-            if converged:
+        if (has_vasp_output and converged) or has_ase_output:
```

Comment:
Likely my fault but I just realized that it is probably rather 
```suggestion
        if has_vasp_output or has_ase_output:
```